elements/bulbs-header/bulbs-header-test.js

```diff
@@ -0,0 +1,47 @@
+import './bulbs-header';
+
+describe('<bulbs-header>', () => {
+  let header;
+  let masthead;
+  let responsiveNav;
+
+  beforeEach((done) => {
+    header = document.createElement('bulbs-header');
+    header.innerHTML = `
+      <bulbs-header-mastead></bulbs-header-mastead>
```

Comment:
Is the `masthead` variable just not used in these tests, because `<bulbs-header-masthead>` is spelled wrong here.